### PR TITLE
tidb-config: add txn-local-latches

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -755,6 +755,20 @@ opentracing.reporter 相关的设置。
 + 单位：MB
 + 类型：Float
 
+## txn-local-latches
+
+与事务锁存相关的配置。当本地事务冲突较多时，建议启用该配置。
```
Comment (cfzjywxk) [inline tidb-configuration-file.md:760]: 实际上没怎么维护，还是不建议启用，这块英文文档也可更新下。
后面可能会废弃掉

Comment (hfxsd) [inline tidb-configuration-file.md:760]:
```suggestion
与事务锁存相关的配置。该参数今后可能会废弃，不建议启用。
```

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -755,6 +755,20 @@ opentracing.reporter 相关的设置。
 + 单位：MB
 + 类型：Float
 
+## txn-local-latches
+
+与事务锁存相关的配置。该参数今后可能会废弃，不建议启用。
```
Comment (qiancai) [inline tidb-configuration-file.md:760]: https://github.com/pingcap/docs/pull/17253#discussion_r1570531413
image

Comment (hfxsd) [inline tidb-configuration-file.md:760]:
```suggestion
与事务锁存相关的配置项。这些配置项今后可能会废弃，不建议启用。
```

**Review by cfzjywxk — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**